tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_BentBeam.cpp

```diff
@@ -0,0 +1,79 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cstddef>
+#include <tuple>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Elliptic/Systems/Elasticity/Tags.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+struct BentBeamProxy : Elasticity::Solutions::BentBeam {
+  using Elasticity::Solutions::BentBeam::BentBeam;
+
+  using field_tags = tmpl::list<Elasticity::Tags::Displacement<2>,
+                                Elasticity::Tags::Stress<2>>;
+  using source_tags =
+      tmpl::list<Tags::Source<Elasticity::Tags::Displacement<2>>>;
+
+  tuples::tagged_tuple_from_typelist<field_tags> field_variables(
+      const tnsr::I<DataVector, 2>& x) const noexcept {
+    return Elasticity::Solutions::BentBeam::variables(x, field_tags{});
+  }
+
+  tuples::tagged_tuple_from_typelist<source_tags> source_variables(
+      const tnsr::I<DataVector, 2>& x) const noexcept {
+    return Elasticity::Solutions::BentBeam::variables(x, source_tags{});
+  }
+};
+
+}  // namespace
+
+SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.AnalyticSolutions.Elasticity.BentBeam",
+    "[PointwiseFunctions][Unit][Elasticity]") {
+  const Elasticity::Solutions::BentBeam check_solution{
+      5., 1., 0.5,
+      // Iron: E=100, nu=0.29
```

Comment:
Sorry, this is unclear at this point because the `IsotropicHomogeneous` constitutive relation is added in #1298. It defines the quantities E and nu in terms of its options K and mu. This PR has a dependence on that.